docs/source/_templates/custom-navbar.html

```diff
@@ -1,6 +1,6 @@
 <ul id="navbar-main-elements" class="navbar-nav">
-    <li class="nav-item ">
-        <a class="nav-link" href="installation.html">Installation</a>
```

Comment:
Updated the installation link to be dynamic as it was failing in docs page if you got API reference page